vault/policy_store.go

```diff
@@ -753,10 +753,11 @@ func (t *TemplateError) Error() string {
 }
 
 // ACL is used to return an ACL which is built using the
-// named policies.
-func (ps *PolicyStore) ACL(ctx context.Context, entity *identity.Entity, policyNames map[string][]string) (*ACL, error) {
-	var policies []*Policy
-	// Fetch the policies
+// named policies and pre-fetched policies if given.
+func (ps *PolicyStore) ACL(ctx context.Context, entity *identity.Entity, policyNames map[string][]string, policies ...*Policy) (*ACL, error) {
```

Comment:
To disambiguate the last two parameters maybe the new param could be called `additionalPolicies` or `inlinePolicies` or something?